app/controllers/utils/ControllerUtils.scala

```diff
@@ -61,16 +62,18 @@ class ControllerUtils @Inject()(configuration: Configuration) extends URIInforma
     formats: json.Reads[Map[String, String]]): Future[Result] =
     wsResponse.map { response =>
       response.status match {
-        case 200 => {
+        case OK => {
           if (response.body.contains("appStatusMessage")) {
             Logger.warn(
-              "GenerateResultBasedOnStatus Response Failed status:" + response.status + " json:" + " body:" + response.body)
-
+              s"[ControllerUtils][generateResultBasedOnStatus] GenerateResultBasedOnStatus Response Failed status: ${response.status}"
+            )
             val msgValue = extractUpstreamError(response.body)
-
             val error = PbikError(msgValue)
-            if (error.errorCode == "63082") Ok(Json.toJson(List[EiLPerson]()))
-            else new Status(response.status)(Json.toJson(error))
+            if (error.errorCode == "63082") {
+              Ok(Json.toJson(List[EiLPerson]()))
+            } else {
+              new Status(response.status)(Json.toJson(error))
+            }
           } else {
             // TODO - why does response.header("eTag") return Null when the Option should.. but Tests fail without it
```

Comment:
Does a story need to be raised for this TODO thingy?

Reply:
possibly, it is 5 years old